src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+        return {
+            brand: null,
+            brands: [
+                { name: 'rock-paper-scissors', text: 'edits films, television, commercials, and music videos.'},
+                { name: 'elastic', text: 'creates animation, motion design, and main titles.'},
+                { name: 'a52', text: 'is a leader in visual effects.'},
+                { name: 'primary', text: 'grades color and manages workflows and telecine.'},
+                { name: 'jax', text: 'markets films, tv shows, and games.'},
+                { name: 'indestructible', text: 'supervises musical direction, composition, and licensing.'},
+            ],
+            timer: null,
+            imagesLength: 0,
+            activeImage: 0
+        }
+    },
+    computed: {
+        images() {
+            switch (this.brand) {
+                case 'rock-paper-scissors':
+                    this.imagesLength = this.rpsImages.length
+                    return this.rpsImages
+                case 'elastic':
+                    this.imagesLength = this.elasticImages.length
+                    return this.elasticImages
+                case 'a52': 
+                    this.imagesLength = this.a52Images.length
+                    return this.a52Images
+                case 'primary':
+                    this.imagesLength = this.primaryImages.length
+                    return this.primaryImages
+                case 'jax': 
+                    this.imagesLength = this.jaxImages.length
+                    return this.jaxImages
+                case 'indestructible': 
+                    this.imagesLength = this.indestructibleImages.length
+                    return this.indestructibleImages
+            }
+        }
+    },
+    methods: {
+        set(brand) {
+            if (!brand) return this.reset()
+            this.brand = brand
+            this.start()
+        },
+        start() {
+            if (!this.timer) {
+                this.timer = setInterval( () => {
+                    return this.activeImage < this.imagesLength
+                        ? this.activeImage = this.activeImage + 1 
+                        : this.activeImage = 0
+                }, 500)
+            }
+        },
+        reset() {
+            clearInterval(this.timer)
+            this.brand = null
+            this.activeImage = 0
+            this.timer = null
+        }
+    }
+}
+</script>
+
+<style lang="scss" scoped>
+.gallery-text {
+    position: relative;
+    z-index: 0;
+    background-color: black;
+    padding: var(--unit-gutter);
+    box-sizing: border-box;
+
+    font-size: 33px;
+    font-weight: 300;
+    color: #d8d9be;
+
+    .subtitle {
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+    }
+    .prompt {
+        position: relative;
+        padding-left: 1rem;
+    }
+    .prompt-text {
+        font-size: 14px;
+        font-weight: 500;
+        overflow: hidden;
+        transition: opacity 200ms ease;
```

Comment:
Do this instead here please, and anywhere else with a transition on it?

```
         transition: opacity 200ms $authenticMotion;
```

That is a nice custom easing we use a lot.